guides/assets/javascripts/guides.js

```diff
@@ -1,53 +1,53 @@
-$.fn.selectGuide = function(guide) {
-  $("select", this).val(guide);
-};
-
-var guidesIndex = {
-  bind: function() {
-    var currentGuidePath = window.location.pathname;
-    var currentGuide = currentGuidePath.substring(currentGuidePath.lastIndexOf("/")+1);
-    $(".guides-index-small").
-      on("change", "select", guidesIndex.navigate).
-      selectGuide(currentGuide);
-    $(document).on("click", ".more-info-button", function(e){
-      e.stopPropagation();
-      if ($(".more-info-links").is(":visible")) {
-        $(".more-info-links").addClass("s-hidden").unwrap();
-      } else {
-        $(".more-info-links").wrap("<div class='more-info-container'></div>").removeClass("s-hidden");
-      }
-    });
-    $("#guidesMenu").on("click", function(e) {
-      $("#guides").toggle();
-      return false;
+(function() {
+  "use strict";
+  window.syntaxhighlighterConfig = { autoLinks: false };
+
+  this.wrap = function(elem, wrapper) {
```

Comment:
`this` is `window` here. Are you intending to make these helpers globally available?